src/main/java/com/microsoft/cse/helium/app/services/keyvault/KeyVaultService.java

```diff
@@ -9,6 +10,8 @@
 import com.microsoft.azure.keyvault.models.KeyBundle;
 import com.microsoft.azure.keyvault.models.SecretBundle;
 import com.microsoft.azure.keyvault.models.SecretItem;
+//import com.microsoft.azure.management.Azure;
+//import com.microsoft.azure.management.keyvault.Vault;
 import com.microsoft.cse.helium.app.Constants;
```

Comment:
is pmd checks letting this? Usually it barfs with ununsed imports

Reply:
:D Yes, as you know, we will have build break if code is not PMD clean. That being said, I just fixed it.